src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs

```diff
@@ -368,7 +364,7 @@ private void ReleaseCaptures(PointerRoutedEventArgs routedArgs)
 		{
 			if (PointerCapture.TryGet(routedArgs.Pointer, out var capture))
 			{
-				foreach (var target in capture.Targets)
+				foreach (var target in capture.Targets.ToList())
```

Comment:
Why?

Reply:
Because the `ReleaseointerCapture` below is expected to modify the `capture.Targets` list, so we need to clone it before.